Doxyfile

```diff
@@ -821,10 +821,10 @@ EXCLUDE_SYMLINKS       = NO
 # Note that the wildcards are matched against the file with absolute path, so to
 # exclude all test directories for example use the pattern */test/*
 
-EXCLUDE_PATTERNS        = */demos/*
-EXCLUDE_PATTERNS       += */examples/*
-EXCLUDE_PATTERNS       += */tests/config/*
-EXCLUDE_PATTERNS       += */third-party/*
+EXCLUDE_PATTERNS        = */demos/* 
```

Comment:
trailing whitespace diffs intentional?

Reply:
They were at one point, but they turn out not to make a difference. I'll get rid of them. I think we're actually not using the Doxyfile at all anymore now that we've switched to Exhale, but I'm kind of scared to get rid of it.